packages/api-client/src/views/Request/RequestSidebar.vue

```diff
@@ -103,6 +103,12 @@ onMounted(() => events.hotKeys.on(handleHotKey))
 onBeforeUnmount(() => {
   events.hotKeys.off(handleHotKey)
 })
+
+const handleToggleWatchForChanges = (item?: SidebarItem) => {
+  if (item && item.watchForChanges !== undefined) {
+    item.watchForChanges = !item.watchForChanges
+  }
+}
```

Comment:
Is it safer to check for the presence of `item.url` and maybe more self-explanatory than `item.watchForChanges !== undefined`? 🤔